packages/integration-test/test/scheduled/eth-input-data.test.ts

```diff
@@ -35,18 +35,18 @@ describe('ETH Fee proxy detection test-suite', () => {
     const balance = await ethInputContract.getBalance({
       ...requestData,
       currency: {
-        network: 'private',
+        network: 'rinkeby',
         type: RequestLogicTypes.CURRENCY.ETH,
```

Comment:
I had to change network from private to rinkeby, because payment detection was throwing errors saying it can't work on private network because it uses a block explorer. 
Is this ok @benjlevesque ?

Reply:
yep